routes/users.js

```diff
@@ -1,4 +1,33 @@
 const express = require('express');
+const donationData = require('../data/donations');
 const router = express.Router();
 
+router.get('/admin/:id', async (req, res) => {
+  let donations = await donationData.allDonations();
+  res.status(200).render('users/admin/dashboard', {
+    title: 'Admin',
+    showApproveReject: true,
+    donations,
+    message: req.flash(),
+  });
+});
+
+router.get('/donor/:id', async (req, res) => {
+  let donations = await donationData.getApprovedDonations();
+  res.status(200).render('users/admin/dashboard', {
```

Comment:
Will change this later when I integrate Recipient and Donor page with all of this..